feathr_project/feathr/registry/_feature_registry_purview.py

```diff
@@ -1368,11 +1368,11 @@ def _get_transformation_from_dict(self, input: Dict) -> FeatureType:
         if 'transformExpr' in input:
             # it's ExpressionTransformation
             return ExpressionTransformation(input['transformExpr'])
-        elif 'def_expr' in input:
-            agg_expr=input['def_expr'] if 'def_expr' in input else None
-            agg_func=input['agg_func']if 'agg_func' in input else None
+        elif 'def_expr' in input or 'defExpr' in input: 
```

Comment:
Just want to confirm will camel and snake format occur together?
I mean, in current fix, the code will still check `def_expr`, `agg_func` and `group_by` if `defExpr` in input.